src/scripts.js

```diff
@@ -22,21 +24,47 @@ const userSteps = document.getElementById('userSteps')
 const viewTeamButton = document.querySelector('#viewTeam');
 const hydrationMenu = document.querySelector('.hydMenu');
 const hydrationWrapper = document.querySelector('.hydrationWrapper')
-const mainBody = document.querySelector('.mainbody')
+const dailySection = document.querySelector('.dailyWidgets')
+const weeklySection = document.querySelector('.weeklyWidgets')
+const allTimeSection = document.querySelector('.allTime')
+const sleepMenu = document.querySelector('.slpMenu')
+const dailyCalendar = document.querySelector('.dayCal')
+const weeklyCalendar = document.querySelector('.weekCal')
 
 // ********** EVENT LISTENERS **********
 window.addEventListener('load', updateUser);
-userName.addEventListener('click', displayProfile);
+userPhoto.addEventListener('click', displayProfile);
 sideBarContents.addEventListener('click', displayTeamList);
-// myProfileLink.addEventListener('click', displayProfile);
-hydrationMenu.addEventListener('click', displayHydrationData)
-
+hydrationMenu.addEventListener('click', displayHydrationData);
+sleepMenu.addEventListener('click', displaySleepData);
+dailyCalendar.addEventListener('input', changeDate);
+weeklyCalendar.addEventListener('input', changeDate);
 // ******** FUNCTIONS/EVENTHANDLERS **********
 
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
 
+function changeDate(event){
+  event.preventDefault()
+  if(event.target === dailyCalendar) {
+    now = moment(`${dailyCalendar.value}`).format('YYYY/MM/DD')
+  } else if(event.target === weeklyCalendar) {
+    weekEnd = moment(`${weeklyCalendar.value}`).format('YYYY/MM/DD')
+  }
+  refreshPageData(event);
+}
+
+function refreshPageData(event) {
+  if(currentPage === 'hydration') {
+    displayHydrationData()
+  } else if(currentPage === 'sleep') {
+    displaySleepData()
+  }
```

Comment:
Obviously add the displayActivityData once you get your hands on it. But you knew that.